crates/matrix-sdk/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 - The `MediaRetentionPolicy` can now trigger regular cleanups with its new
   `cleanup_frequency` setting.
   ([#4603](https://github.com/matrix-org/matrix-rust-sdk/pull/4603))
+- The HTTP client only allows TLS 1.2 or newer, as recommended by BCP 195.
```

Comment:
I think it should be marked as a breaking change. Plus, can you add a link to the PR + a link to the BCP please?

Reply:
Done